### PR TITLE
Fix invalid cells cached on infinite loaded grids

### LINKED ISSUE #526: InfiniteScroll + cellCache behavior
Smaller bug since it's so easy to get around by customizing the `cellRangeRenderer`, but I noticed that while scrolling a grid that's wrapped in an InfiniteScroll, cells that were not loaded when the scrolling started do not get a chance to update when the data is loaded until after the `scrollingResetTimeInterval` delay.

grid

### PR DESCRIPTION
**Pull request by nickclaw**
Closes #526

### CODE REVIEW

File: source/InfiniteLoader/InfiniteLoader.test.js [hunk 1]
```diff
@@ -360,4 +360,36 @@ describe('forceUpdateReactVirtualizedComponent', () => {
...
+    forceUpdateReactVirtualizedComponent(
+      render(<TestComponent />)
+    )
+    expect(recomputedGridSize).toEqual(true)
+  })
+
+  it('should called :recomputeRowHeights if defined', () => {
+    let recomputedRowHeights = false
+    class TestComponent extends Component {
+      forceUpdate () {
```
Comment (bvaughn) [inline source/InfiniteLoader/InfiniteLoader.test.js]: This method should be named `recomputedRowHeights`

Comment (nickclaw) [inline source/InfiniteLoader/InfiniteLoader.test.js]: Good catch, will update.

**Comment by nickclaw** [comment]
Yeah, with this PR and the one just merged my grid is crazy smooth

**Comment by bvaughn** [comment]
Fantastic. Thank you!

**Comment by bvaughn** [comment]
Soon as that method is renamed in the test I'll merge.

**Comment by bvaughn** [comment]
Thank you! :)

**Pull request merged**